### PR TITLE
Added section for past events. Added Berlin code of conduct. Added De…

### PR DESCRIPTION
**Pull request by Evie-Skinner18**
…cember event to top

Solves #3 and #10 
Adds December event to the site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,49 +38,72 @@ <h1>Queer Tech Bristol</h1>
...
+                <img
+                  class="events__image"
+                  alt="a dithered close up image of Cliff Richard riding a sled with his fist in the air"
+                  src="./images/queermas_cliff.png"/>
+              </div>
+            </li>
+          </ul>
+        </section>
+
+        <section class="past-events">
```
Comment (Harriethw) [inline index.html]: it looks like the `past-events` and `current-events` classes aren't being used, did you mean to add some styling to them or did you mean them to be more semantic? If they are semantic or for accessibility, I think something like an `aria-label` or `aria-labelledby` tag makes more sense https://www.w3.org/WAI/GL/wiki/Using_HTML5_section_element 

If you *did* want to add styles, [please can we use BEM naming guidelines](https://getbem.com/introduction/) for css? so something like `events__past` maybe?

File: index.html [hunk 2]
```diff
@@ -38,49 +38,72 @@ <h1>Queer Tech Bristol</h1>
...
+              </div>
+              <div>
+                <img
+                  class="events__image"
+                  alt="a dithered image of  a rose"
+                  src="./images/showcase.gif"/>
+              </div>
+            </li>
+            <li class="events__event">
+              <div class="events__description">
```
Comment (Harriethw) [inline index.html:98]: I think this `div` needs to be closed after line 106

**Review by Harriethw — CHANGES_REQUESTED** [review]
thanks so much Evie, can't wait to jingle some bells with you! just had a few comments :) 

The styling on the past events looks a bit off for the second event so think something has been overwritten or lost there 
Screenshot 2022-11-28 at 13 51 02

Alsooo this is a small thing and shouldn't stop merging but while I'm here - I wondered if you'd seen the GDS styleguide for commits? One thing they suggest is writing commit messages in present tense as this matches commands generated by stuff like `git merge` (and keeps them shorter!)  https://github.com/alphagov/styleguides/blob/master/git.md#example-1

**Comment by Evie-Skinner18** [comment]
Thanks for the tips Hazzer @Harriethw I've addressed your feedback

**Review by Harriethw — APPROVED** [review]

**Pull request merged**